tests/unit/conftest.py

```diff
@@ -245,6 +246,14 @@ def oauth1_service(pyramid_config):
     return oauth1_service
 
 
+@pytest.fixture
+def token_store_service(oauth_token, pyramid_config):
+    token_store_service = mock.create_autospec(TokenStore, instance=True, spec_set=True)
+    token_store_service.get.return_value = oauth_token
+    pyramid_config.register_service(token_store_service, name="token_store")
+    return token_store_service
```

Comment:
Centralised service fixture, as we have for all the services. This means we only need one of these fixtures and tests for all code that uses `TokenStore` can use this fixture